presto-parser/src/test/java/com/facebook/presto/sql/parser/TestSqlParser.java

```diff
@@ -1413,6 +1423,64 @@ public void testCreateView()
         assertStatement("CREATE VIEW \"awesome schema\".\"awesome view\" AS SELECT * FROM t", new CreateView(QualifiedName.of("awesome schema", "awesome view"), query, false));
     }
 
+    @Test
+    public void testCreateFunction()
+    {
+        assertStatement(
+                "CREATE FUNCTION tan (x double)\n" +
+                        "RETURNS double\n" +
+                        "SQL\n" +
+                        "DETERMINISTIC\n" +
```

Comment:
Add test for default `NOT DETERMINISTIC` and `RETURNS NULL ON NULL INPUT`.

Reply:
Line 1464-1468 covers those cases.